src/spork.cpp

```diff
@@ -88,8 +88,7 @@ void CSporkManager::CheckAndRemove()
             ++itSignerPair;
         }
         if (itActive->second.empty()) {
-            mapSporksActive.erase(itActive++);
-            continue;
+            mapSporksActive.erase(itActive);
```

Comment:
Wym? This should be a pure refactor... I don't see how it changes behavior in any way. Just instead of incrementing on L91 and then skipping L93, we don't increment on L91, and don't skip L93, so increment on L93

Reply:
With `itActive++` you increment it before `erase`, with your patch you try to access it after `erase` which is pretty much an undefined behaviour afaik.